<launch>
    <arg name="events" default="true" />
    <arg name="sm_gui" default="false" />
    <arg name="reconf_gui" default="false" />

    <!-- task1 node -->
    <node name="task1" pkg="src_task1" type="src_task1_node" output="screen"/>

    <!-- state transition visualisation on rviz -->
    <node
    if="$(arg sm_gui)"
    name="rqt" pkg="rqt_gui" type="rqt_gui" args="-s rqt_decision_graph" output="screen" />


    <!-- Start up the dynamic reconfigure GUI. -->
    <node
    if="$(arg reconf_gui)"
    pkg="rqt_reconfigure" type="rqt_reconfigure" name="rqt_reconfigure"/>

    <!-- Start up rqt_console -->
    <!--node
    if="$(arg reconf_gui)"
    pkg="rqt_console" type="rqt_console" name="rqt_console"/-->

</launch>
